openlibrary/i18n/de/messages.po

```diff
@@ -7,2187 +7,5520 @@ msgid ""
+#: type/edition/view.html:190 type/work/view.html:190
+msgid "by"
+msgstr "von"
+
+#: diff.html:44
+msgid "by Anonymous"
+msgstr "von Anonym"
+
+#: diff.html:110
+msgid "Differences"
+msgstr "Änderungen"
+
+#: diff.html:168
+msgid "Both the revisions are identical."
+msgstr "Beide Versionen sind identisch."
+
+#: edit_yaml.html:14 lib/edit_head.html:17
+msgid "You're in edit mode."
+msgstr "Sie sind im Bearbeitungsmodus."
+
+#: edit_yaml.html:15 lib/edit_head.html:18
+msgid "Cancel, and go back."
+msgstr "Abbrechen und zurück."
+
+#: edit_yaml.html:22
+msgid "YAML Representation:"
+msgstr "YAML Darstellung:"
+
+#: BookPreview.html:11 editpage.html:15
+msgid "Preview"
+msgstr "Vorschau"
+
+#: history.html:23
+msgid "History of edits to"
+msgstr "Änderungsverlauf zu"
+
+#: history.html:31
+msgid "Revision"
+msgstr "Version"
+
+#: RecentChanges.html:17 RecentChangesAdmin.html:20 RecentChangesUsers.html:20
+#: admin/ip/view.html:44 admin/people/edits.html:41 history.html:32
+#: recentchanges/render.html:30 recentchanges/updated_records.html:28
+msgid "When"
+msgstr "Wann"
+
+#: RecentChanges.html:19 admin/ip/view.html:46 admin/loans_table.html:46
+#: admin/people/edits.html:43 admin/waitinglists.html:28 history.html:33
+#: recentchanges/render.html:32
+msgid "Who"
+msgstr "Wer"
+
+#: RecentChanges.html:20 RecentChangesUsers.html:22 admin/ip/view.html:47
+#: admin/people/edits.html:44 history.html:34 recentchanges/render.html:33
+#: recentchanges/updated_records.html:30
+msgid "Comment"
+msgstr "Kommentieren"
+
+#: history.html:35
+msgid "Diff"
+msgstr "Änderung"
+
+#: history.html:42 history.html:73
+msgid "Compare"
+msgstr "Vergleichen"
+
+#: history.html:42 history.html:73
+msgid "See Diff"
+msgstr "Änderungen ansehen"
+
+#: history.html:48 lib/history.html:78
+#, python-format
+msgid "View revision %s"
+msgstr "Version %s anschauen"
+
+#: history.html:85
+msgid "Back"
+msgstr "Zurück"
+
+#: Pager.html:34 history.html:87 lib/pagination.html:37 showmarc.html:54
+msgid "Next"
+msgstr "Nächste"
+
+#: internalerror.html:14
+msgid "Sorry. There seems to be a problem with what you were just looking at."
+msgstr "Verzeihung. Es scheint ein Problem mit der Seite zu geben."
+
+#: internalerror.html:15
+#, python-format
+msgid ""
+"We've noted the error %s and will look into it as soon as possible. Head for <a "
+"href=\"/\">home</a>?"
+msgstr ""
+"Wir haben den Fehler %s aufgenommen und werden das Problem schnellstmöglich "
+"bearbeiten. Zur <a href=\"/\">Startseite</a> gehen?"
+
+#: lib/nav_head.html:30 library_explorer.html:6
+msgid "Library Explorer"
+msgstr "Bibliotheksexplorer"
+
+#: lib/header_dropdown.html:46 lib/nav_head.html:107 login.html:10 login.html:72
+msgid "Log In"
+msgstr "Anmelden"
+
+#: login.html:15
+msgid ""
+"Please enter your <a href=\"https://archive.org\">Internet Archive</a> email and "
+"password to access your Open Library account."
+msgstr ""
+"Bitte geben Sie Ihr <a href=„https://archive.org“>Internet Archive</a> E-Mail-"
```

Comment:
```suggestion
"Bitte geben Sie Ihr <a href=\"https://archive.org\">Internet Archive</a> E-Mail-"
```
Inside of HTML tags, replace opening and closing double quote symbols with the straight double quote symbol